client/src/components/Tool/ToolForm.vue

```diff
@@ -0,0 +1,320 @@
+        FormElement,
+        ToolSuccess,
+        Webhook,
+    },
+    props: {
+        id: {
+            type: String,
+            default: null,
+        },
+        version: {
+            type: String,
+            default: null,
+        },
+        job_id: {
+            type: String,
+            default: null,
+        },
+        history_id: {
+            type: String,
+            default: null,
+        },
+    },
+    data() {
+        return {
+            showLoading: true,
+            showForm: false,
+            showEntryPoints: false,
+            showRecommendation: false,
+            showSuccess: false,
+            showError: false,
+            showExecuting: false,
+            error: null,
+            formConfig: {},
+            formData: {},
+            errorTitle: "",
+            errorContent: null,
+            messageVariant: "",
+            messageText: "",
+            reuse: false,
+            email: false,
+            remap: false,
+            entryPoints: [],
+            jobDef: {},
+            jobResponse: {},
+            validationInternal: null,
+            validationScrollTo: null,
+            validationErrors: null,
+            currentVersion: this.version,
+        };
+    },
+    created() {
+        this.requestTool();
+        const Galaxy = getGalaxyInstance();
+        if (Galaxy.currHistoryPanel) {
+            Galaxy.currHistoryPanel.collection.on("change", () => {
+                this.onUpdate();
+            });
+        }
+    },
+    computed: {
+        toolName() {
+            return this.formConfig.name;
+        },
+        tooltip() {
+            return `Execute: ${this.formConfig.name} (${this.formConfig.version})`;
+        },
+        errorContentPretty() {
+            return JSON.stringify(this.errorContent, null, 4);
+        },
+        inputs() {
+            return this.formConfig.inputs;
+        },
+        remapAllowed() {
+            return this.job_id && this.formConfig.job_remap;
+        },
+        remapLabel() {
+            if (this.formConfig.job_remap === "job_produced_collection_elements") {
+                return "Replace elements in collection ?";
+            } else {
+                return "Resume dependencies from this job ?";
+            }
+        },
+        remapHelp() {
+            if (this.formConfig.job_remap === "job_produced_collection_elements") {
+                return "The previous run of this tool failed. Use this option to replace the failed element(s) in the dataset collection that were produced during the previous tool run.";
+            } else {
+                return "The previous run of this tool failed and other tools were waiting for it to finish successfully. Use this option to resume those tools using the new output(s) of this tool run.";
+            }
+        },
+    },
+    methods: {
+        emailAllowed(config, user) {
+            return config.server_mail_configured && !user.isAnonymous();
```

Comment:
It appears to be not a function now, but a value

```suggestion
            return config.server_mail_configured && !user.isAnonymous;
```

Reply:
Yes, this has been resolved.

Reply:
@guerler can you take a look in your gitter please? :)